python/requirements/test-requirements.txt

```diff
@@ -114,4 +115,4 @@ sympy==1.10.1; python_version <= '3.7'
 
 # For test_basic.py::test_omp_threads_set
 threadpoolctl==3.1.0
-numexpr==2.8.4
+numexpr==2.8.4
```

Comment:
nit: what changed on this line? eof eol?

Reply:
the old file has a space at the end of the string

Reply:
actually i don't even know ;)